Drv/LinuxUartDriver/LinuxUartDriverCommon.cpp

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
If we're not bothering to have stubs, this could be combined with `LinuxUartDriver.cpp` to keep the file count lower.